ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -369,6 +375,56 @@ public sealed class ArchiWebHandler : IDisposable {
 			return result;
 		}
 
+		[PublicAPI]
+		public async Task<int?> GetPointsBalance() {
+			(bool success, string? accessToken) = await CachedAccessToken.GetValue().ConfigureAwait(false);
+
+			if (!success || string.IsNullOrEmpty(accessToken)) {
+				return null;
+			}
+
+			KeyValue? response = null;
+
+			for (byte i = 0; (i < WebBrowser.MaxTries) && (response == null); i++) {
+				using WebAPI.AsyncInterface loyaltyRewardsService = Bot.SteamConfiguration.GetAsyncWebAPIInterface(LoyaltyRewardsService);
+
+				loyaltyRewardsService.Timeout = WebBrowser.Timeout;
+
+				try {
+					response = await WebLimitRequest(
+						WebAPI.DefaultBaseAddress,
+
+						// ReSharper disable once AccessToDisposedClosure
+						async () => await loyaltyRewardsService.CallAsync(
+							HttpMethod.Get, "GetSummary", args: new Dictionary<string, object>(2, StringComparer.Ordinal) {
```

Comment:
Since we need to account for SteamRE/SteamKit#992, consider call `.ToDictionary` on created argument dictionary, you can take [this commit](https://github.com/JustArchiNET/ArchiSteamFarm/commit/4a9c8b59d6f62fce8958cca3fae233525af61d4b) as an example.

Reply:
I was about to do it myself after PR but yeah, this could be done in advance.